src/utils/config.ts

```diff
@@ -97,17 +98,26 @@ export class Configuration implements vsc.Disposable {
     }
 
     private async loadSingleSnippet(url: string) {
-        if (url.startsWith('http')) {
+        if (url.startsWith("http") || url.startsWith("https")) {

```

Comment:
> Don't you think `https` also starts with `http` hence this change is reduntant? 😉
🤦 let's say it was a long week


> Also please keep to single quotes. I know it's a mess in this repo but my intent was to use single quotes (like all my other JS/TS repos).

Indeed. Just tried running a format, my prettier kicked in and it messed the whole file up.

> I have just added a ES Lint rule to actually report that as a warning and also fixes all quotes all over the repo.
> Please update your branch to see the changes.

Done

> I wonder how you use it. Can you elaborate a bit?

Well, since it currently wasn't possible yet to have them relatively specified, I'm still experimenting what's the best way to use it. But what I missed up to this point, is how to share these snippets with my team without having to bother them with adding configuration everytime.

> But it will only work if planner file will be saved in root of the repo, right? Otherwise relative path won't work.
You have the flexibility to open up your code anywhere in your repo, but indeed, you'd have to configure a vscode settings somewhere. Suppose you have a mono repository, you could have parts in your repo having their own snippets, but you still need to open up that specific folder in your vscode.

Currently we're adding the snippets to the root of the project in a .planner folder and opening up code in this root folder. It also allows us to quickly peek at a part of the code if there is some question or if something is not clear to someone.
But another option is to open up this only in the .planner folder and add a .vscode folder and settings file there to specify the snippets and have all planning related stuff in that folder. Still have to experiment what the best way is, but I think this depends on the team.

> You want to just include those snippets in project repo so that the team can use it to plan stories?
Indeed. 
- Share it with the team: Given the point I was struggling with like I said above, this would mean my whole team has access to the snippets, and configuration is done only once, and everyone can use them on the next pull. Another advantage is if the team decides to update the tasks and/or hours, then everyone has the updated snippets immediately. 
- Repo specific timings: Depending on the project the team is working on, these could vary in tasks and/or hours. So keeping them with the repo and project seems to be a good idea.

Note that we currently have a single repo per project and we usually only work on 1 project in a 2-week sprint.

> Do you have a habit of doing so?
I'd like to have as much information around the development process in source control. Just like we have our development flow (PR (with or without templates) -> master -> release ) also integrated in it, why not add your planning needs as well. We could technically store it in another location, but why should we if integrating automatically with our tools is possible.
At the moment we already have our planning settings specified in the workspace settings file (excluding the PAT). So if needed, I can change to any team in our organization just by opening up their repository and start planning away. If the relative paths are added, I can even start using their snippets without touching any configuration, only requirement is to open the right folder with vscode settings.

Reply:
> > Don't you think `https` also starts with `http` hence this change is reduntant? 😉
> 
> 🤦 let's say it was a long week

It happens, no worries ;)

> > Also please keep to single quotes. I know it's a mess in this repo but my intent was to use single quotes (like all my other JS/TS repos).
> 
> Indeed. Just tried running a format, my prettier kicked in and it messed the whole file up.

Good point. Since prettier is pretty much industry standard now I guess I need to think of putting a config for it so that it doesn't break in the future. 

Thanks for explaining your use case. I didn't even think of that when I was first using that extension but that makes a lot of sense! That's really good to know how people use that. When I'm on my next project which uses Azure DevOps I might want to introduce the same. Last time when I was using it we have been sharing the snippets via GH Gist or similar but that of course requires configuration for each team member. 
Sharing snippets along with configuration sounds like a way to go given that most of the team uses the extension. 
I'll remember that. I'm in JIRA world now so it's totally different :)